charts/airflow/files/pod_template.kubernetes-helm-yaml

```diff
@@ -74,6 +74,10 @@ spec:
       volumeMounts:
         {{- $volumeMounts | indent 8 }}
       {{- end }}
+    
```

Comment:
nit: can you please remove this newline.

Reply:
removed